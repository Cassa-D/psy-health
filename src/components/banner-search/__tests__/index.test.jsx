import React from "react";
import renderer from "react-test-renderer";

import BannerSearch from "../index";

it("render correctly BannerSearch", () => {
  const tree = renderer.create(<BannerSearch />).toJSON();

  expect(tree).toMatchSnapshot();
});
