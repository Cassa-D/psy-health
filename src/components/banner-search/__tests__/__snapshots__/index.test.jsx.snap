// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly BannerSearch 1`] = `
<div
  className="sc-gtssRu iousAW"
>
  <div
    className="sc-dlnjPT hkNVgF"
  >
    <img
      className="sc-hKFyIo faMfon"
      src="img_banner_search.svg"
    />
  </div>
  <div
    className="sc-eCApGN gwYTeE"
  >
    <div
      className="sc-jSFkmK jrMmYd"
    >
      <h3
        className="sc-iCoHVE gcvEML"
      >
        Encontre o psicólogo ideal pra você
      </h3>
      <button
        className="sc-bdnylx jsFNV"
        data-text="Buscar agora"
        height="60px"
        onClick={[Function]}
        width="200px"
      >
        <span>
          B
        </span>
        <span>
          u
        </span>
        <span>
          s
        </span>
        <span>
          c
        </span>
        <span>
          a
        </span>
        <span>
          r
        </span>
        <span>
           
        </span>
        <span>
          a
        </span>
        <span>
          g
        </span>
        <span>
          o
        </span>
        <span>
          r
        </span>
        <span>
          a
        </span>
      </button>
    </div>
  </div>
</div>
`;
